client/src/modules/patients/groups/modal/createUpdate.html

```diff
@@ -0,0 +1,90 @@
+<form name="patientGroupForm" bh-submit="ModalCtrl.submit(patientGroupForm)" novalidate>
+  <div class="modal-header">
+    <ol class="headercrumb">
+      <li>
+        <span translate>PATIENT_GROUP.TITLE</span>
+
+        <label class="label label-warning text-uppercase" translate>
+          {{ModalCtrl.action}}
+        </label>
+      </li>
+    </ol>
+  </div>
+
+  <div class="modal-body">
+    <div class="row">
+        <div class="form-group" ng-class="{ 'has-error' : patientGroupForm.$submitted && patientGroupForm.name.$invalid }">
```

Comment:
Could you wrap this `<div class="form-group"...` inside a `<div class="col-"` according the correct Bootstrap structure of rows and columns.